### PR TITLE
Adds Shibu's post on new LinkedIn group

### PR DESCRIPTION
**Pull request by abhilashlr**

### CODE REVIEW

File: source/2020-06-19-the-ember-times-issue-153.md [hunk 1]
```diff
@@ -115,14 +116,11 @@ Each developer's story is **unique** and **insightful** and can inspire _you_ to
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [New LinkedIn group EmberJS developers 🐹](https://www.linkedin.com/posts/shibulijack_ember-community-strongertogether-activity-6673898303190654976-BgRg/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Shibu Lijack](https://github.com/shibulijack) created a LinkedIn **group for EmberJS enthusiasts**. It is a group that welcomes anyone who wants to share the latest and greatest of EmberJS that is shared on [EmberJS blogs](https://blog.emberjs.com/) and the [EmberJS Discord](https://discord.com/invite/emberjs) channels.
```
Comment (ijlee2) [inline source/2020-06-19-the-ember-times-issue-153.md]:
```suggestion
[Shibu Lijack (@shibulijack)](https://github.com/shibulijack) created a LinkedIn **group for EmberJS enthusiasts**. It is a group that welcomes anyone who wants to share the latest and greatest of EmberJS that is shared on [EmberJS blogs](https://blog.emberjs.com/) and the [EmberJS Discord](https://discord.com/invite/emberjs) channels.
```

**Review by ijlee2 — APPROVED** [review]
Looks great! I think we could add Shibu's GitHub handle (just a tradition that we've been following; Twitter handle works too), in case people would like to follow him on GitHub.

**Comment by abhilashlr** [comment]
> Looks great! I think we could add Shibu's GitHub handle (just a tradition that we've been following; Twitter handle works too), in case people would like to follow him on GitHub.

Oh yes I missed it this time @ijlee2. I’ll add it.

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for another awesome :sparkles: writeup for the Ember Times :hamster:

**Pull request merged**